src/polyswarmd/bounties.py

```diff
@@ -247,24 +247,35 @@ def post_bounties_guid_assertions(guid):
 
     bid = int(body['bid'])
     mask = bool_list_to_int(body['mask'])
-    verdicts = bool_list_to_int(body['verdicts'])
+
+    commitment = int(body.get('commitment'))
```

Comment:
If commitment isn't provided, I believe this will throw an exception. It will also never be None later.

Reply:
Should be fixed, thx